spec/support/helpers/custom_devise_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CustomDeviseHelper
```

Comment:
do we still need this? not using `login_as` instead? as discussed in previous PR